.github/workflows/publish-crate.yml

```diff
@@ -0,0 +1,73 @@
+name:   Publish Crate
+
+on:
+    push:
+        tags:
+            - tree-hash-v*
+            - tree-hash-derive-v*
+            - eth2-ssz-v*
+            - eth2-ssz-derive-v*
+            - eth2-ssz-types-v*
+            - eth2-serde-util-v*
```

Comment:
This is a dependency for `ssz_types` so I added the `eth2` prefix here for consistency, but let me know if anyone thinks it should be something else